style.css

```diff
@@ -1 +1,190 @@
-/*  Add your styling here */
+/* Add your styling here */
+body{
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+}
+
+.logoup{
+    display: flex;
+    justify-content: center;
+    margin: auto;
+    width: 30%;
+}  
+
+/* cart icon */
+h3{
+    color: #ffffff;
+    float: right;
+    font-size: 16px;
+    padding: 0px 15px 0px 0px;
+
+}
+p{
+    display: flex;
+    justify-content: center;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 16px;
+    line-height: 1.5;
+    color:rgb(245, 245, 245);
+    background-color: rgb(17, 18, 19, 0.5);
+    padding: 0px 20px 0px 20px;
+}
+ul li{
+    float: left;
+    list-style: none;
+    position: relative;
+
+}
+a{
+   display: block; 
+   text-decoration: none;
+   color:white;
+   font-size: 22px;
+}
+ul li ul{
+    display: none;
+    position: absolute;
+    background-color:rgb(63, 8, 8,.8);
+    opacity: px;
+    padding: 20px;
+    border-radius: 0px 0px 5px 5px;
+}
+ul li:hover ul{
+    display: block;
+}
+ul li ul li{
+    width: 120px;
+}
+ul li ul li a{
+    padding: 15px 0px;
+}
+ul li ul li a:hover{
+   background-color: rgb(10, 10, 10, .7); 
+}
+body{
+    background-image: url(../images/cake1.jpg);
+    /* background-position: center bottom; */
+    background-size: cover;
+    background-repeat: no-repeat;
+    /* background-position-x: center; */
+    /* background-position-y: bottom; */
+    overflow: auto;
+}
+ /* table */
+ table{
+     margin: 60px 0px 50px 50px;
+ }
+ th{
+     display: flex;
+ }
+img{
+   width: 100px;
+    height: 90px;
+    display: flex;
+}
+tr{
+    color: white;
+    display: flex;
+    justify-content: center;
+    padding: 20px 0px;
+    font-size: 30px;
+}
+.iframe-video{
+    position: absolute;
+    width: 100%;
+    padding-bottom: 50%;
+    height: 0;
+}
+.iframe-video iframe{
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    padding-bottom: 50%;
+}
+/* footer */
+.content{
+    display: flex;
+    flex-direction: column;
+    color: maroon;
+}
+.facebook{
+    text-align: center;
+    color:red;
+    font-size: 12px;
+}
+
+
+/* MEDIA QUARIES */
+
+@media (min-width: 768px){
+    .logoup{
+        display: flex;
+        justify-content: center;
+        margin: auto;
```

Comment:
for media queries, you only have to state the properties and html/classes that will be overidden, eg. so these three do not have to be repeated as they're already the same in your general css